README.md

```diff
@@ -37,6 +37,21 @@ val gson: Gson = GsonBuilder()
 
 If you already have other type adapter factories registered, be sure place `KotlinReflectiveTypeAdapterFactory` at the beginning of the list.
 
+When calling `create`, an optional `enableDefaultPrimitiveValues` parameter is available (default
+ is `false`). When this property is set to `true`, nonnull primitives will automatically receive
+ default values. The table below shows which primitives are supported.
```

Comment:
Should we mention that they receive default values when none are supplied by the JSON response?

Reply:
Technically, the default values are used to override null. So even if the JSON explicitly sets null to a value, we override it.